python/lsst/ts/ATWhiteLightSource/wlsComponent.py

```diff
@@ -90,7 +90,7 @@ def checkStatus(self):
         KiloArcStatus = namedtuple('KiloArcStatus', ['wattage','greenLED','blueLED','redLED','errorLED'])
         
         voltages = self._readVoltage()
-        cutoff = 3.0
+        cutoff = 2.0
```

Comment:
Hard coded parameter..

Reply:
I talked to patrick about this one and he said it's ok to leave it as a non-configurable param since this is a hard-wired feature of the kiloarc. Still, it's an easy change if we want it.